src/V3AstNodeExpr.h

```diff
@@ -5742,6 +5742,7 @@ class AstVarXRef final : public AstNodeVarRef {
     string m_name;
     string m_dotted;  // Dotted part of scope the name()'ed reference is under or ""
     string m_inlinedDots;  // Dotted hierarchy flattened out
+    bool m_containsGenBlock = false;  // Contains gen block reference
```

Comment:
Please update AstVarXRef::dump to print [GENBLK]; AstVarXRef::dumpJSON (and --gold test updates as needed).